### PR TITLE
/PWGGA/GammaConv NonLinearity Options changed/added for DCal

### PR DESCRIPTION
**Pull request by AMechler**
case 21  PCM-DCal    Pow     for  LHC16k5a+LHC17e2, LHC16k5b
case 22  DCal              Pow     for  LHC16k5a+LHC17e2, LHC16k5b
case 11  PCM-DCal    Const  for  LHC16k5a, LHC16k5b
case 11  PCM-DCal    kSDM  for  LHC17e2
case 12  DCal              Const   for  LHC16k5a+LHC17e2
case 12  DCal              kSDM   for  LHC16k5b

### CODE REVIEW

File: PWGGA/GammaConv/AliCaloPhotonCuts.cxx [hunk 1]
```diff
@@ -4367,10 +4367,11 @@ void AliCaloPhotonCuts::ApplyNonLinearity(AliVCluster* cluster, Int_t isMC)
             energy /= FunctionNL_kSDM(energy, 0.972156, -4.10515, -0.381273);
             energy /= FunctionNL_kSDM(energy, 0.979999, -4.39136, -0.102332);
           }
-          if(fClusterType==3) energy /= FunctionNL_kSDM(energy, 0.980211, -4.374598, -0.171988);
+          if(fClusterType==3 && fCurrentMC==k16k5a) energy /= 0.9870110951;
+          if(fClusterType==3 && fCurrentMC==k17e2) energy /= FunctionNL_kSDM(energy0.986513, 0.430032, -10.99999);
```
Comment (dmuhlhei) [inline PWGGA/GammaConv/AliCaloPhotonCuts.cxx]: please try to do a local compilation before pushing - this will crash for sure

Comment (AMechler) [reply PWGGA/GammaConv/AliCaloPhotonCuts.cxx]: I'm really sorry for this typo, I'll fix it and compile it

Comment (AMechler) [reply PWGGA/GammaConv/AliCaloPhotonCuts.cxx]: i pushed an fix

**Comment by dmuhlhei** [comment]
+1

**Pull request merged**